puppet/check_puppet.rb

```diff
@@ -17,7 +17,8 @@
 require 'optparse'
 require 'yaml'
 
-statedir = "/var/lib/puppet/state"
+puppet_maj_version = `puppet --version`[0].to_i
```

Comment:
Also strikes me that shelling to puppet is just not needed when all we have is 2 options:

```
if File.directory?("/opt/.....")
  statedir = "/opt/...."
elsif File.directory?("/var/....")
  statedir = "/var/...."
end
```

much faster and achieves pretty much the same, since this is just the default and people can change it via a option I think this is fine, especially with puppet 3 now EOL.